doc/install.rst

```diff
@@ -0,0 +1,63 @@
+Installation
+============
+pyXLIGHT is a Python wrapper for the Fortran-based XFOIL code.
+Building and installing the code is a multi-step process that is machine specific.
+This guide goes through the required steps to compile and install pyXLIGHT.
+
+
+Requirements
+------------
+To compile the required XFOIL core components, a Fortran and a C compiler must be installed on your system.
+This can be GNU / GFortran or Intel, default configuration files are packaged for both types of compilers.
+
+In addition to standard compilers, pyXLIGHT requires the following dependencies:
+
+=================== =======
+Package             Version
+=================== =======
+Python              3.X.X
+NumPy               ---
+Sphinx              ---
+sphinx-mdolab-theme ---
+=================== =======
+
+With the exception of Python, which must be installed on your system, these packages can be installed using:
+
+.. code-block:: bash
+
+    $ pip install numpy sphinx sphinx-mdolab-theme
```

Comment:
Again should remove documentation-related packages here since they're not needed to run pyXLIGHT.